### PR TITLE
Basic cypress tests

### PR DESCRIPTION
**Pull request by Siegrift**
Created basic e2e test for most important in app flows.

Hopefully the tests are easy to read (I tried to make them as resilient as I could). Tests require running app and I recommend running it with clear ganache (doing `redeploy` on a server is enough).

Apart from that, I experience flakiness, and I am not sure what is the reason:
- [x] - First, the problem was that our app (at least in dev) is slow. I fixed this by increasing the cypress timeout.
- [x] - Sometimes, when I register a domain. I still cannot manipulate it (e.g. I get `You are not an owner of this domain`) in `/manage-names` subroute. The reason was that we are skipping some characters from time to time
- [x] - Also, some parts of the UI don't play nice with cypress, so I had to use `click({force: true})`. This is because, when we have a list of offerings, there is an invisible element acting as overlay. It has no impact on UX, but it confuses cypress.
- [x] - There were also some interactions (e.g. collapsing, growing animations) that caused cypress to interact with an element too soon. This was fixed by making sure the elements are visible.
- [x] - There are some disabled tests. Rebasing to master didn't help as there is still a bug in the UI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,4 +178,4 @@ node server/name-bazaar.js
 The purpose of linting and formatting is such that the code is readable and consistent.
 
 We use [ethlint](https://github.com/duaraghav8/Ethlint) for linting solidity files. You
-can use `lein npm ethlint` and `lein npm ethlint-fix` to run the linter.
\ No newline at end of file
+can use `lein npm run ethlint` and `lein npm run ethlint-fix` to run the linter.
```
Comment (ladislavdubravsky) [inline README.md:180]: :+1:

**Comment by ladislavdubravsky** [comment]
I'm getting:
```
**Title:** Error running plugin

**Message:** The following error was thrown by a plugin. We stopped running your tests because a plugin crashed. Please check your plugins file (`/home/ladislav/prj/NameBazaar/name-bazaar/cypress/plugins/index.js`)

**Details:** Error: ENOENT: no such file or directory, stat '/initrd.img'

**Stack trace:**
Error: The following error was thrown by a plugin. We stopped running your tests because a plugin crashed. Please check your plugins file (`/home/ladislav/prj/NameBazaar/name-bazaar/cypress/plugins/index.js`)
    at Object.get (/home/ladislav/.cache/Cypress/6.4.0/Cypress/resources/app/packages/server/lib/errors.js:966:15)
    at EventEmitter.handleError (/home/ladislav/.cache/Cypress/6.4.0/Cypress/resources/app/packages/server/lib/plugins/index.js:159:20)
    at EventEmitter.emit (events.js:315:20)
    at ChildProcess.<anonymous> (/home/ladislav/.cache/Cypress/6.4.0/Cypress/resources/app/packages/server/lib/plugins/util.js:19:22)
    at ChildProcess.emit (events.js:315:20)
    at emit (internal/child_process.js:876:12)
    at processTicksAndRejections (internal/process/task_queues.js:85:21)
```
Will look at it in more depth later, but do you have an idea by any chance?

**Comment by Siegrift** [comment]
No, this doesn't happen to me.

**Comment by Siegrift** [comment]
Were you able to debug that error?

**Comment by Siegrift** [comment]
I found out that I experience similar issue, which is caused by how `node` resolves dependencies. It is described in here: https://github.com/marmelab/react-admin/issues/5417#issuecomment-716099254

TLDR: It was caused by [this line](https://github.com/vacuumlabs/name-bazaar/pull/37/commits/479681bae91e0899407d47dea04f77734a76c78e#diff-1b1a73ba561eab6738c8b62510feed5b10edcd25c56626ac79017752b806439dR28) where I imported `cypress-wait-until` dependency, which I later didn't end up using and removed from `project.clj`. Tests were still passing, because I had it installed locally. But you didn't and node wasn't able to find this dependency and ufortunately crashed :D 

Anyway, this is (will be) fixed in last commit.

File: cypress/fixtures/domains.json [hunk 2]
```diff
@@ -0,0 +1,201 @@
+[
```
Comment (Siegrift) [inline cypress/fixtures/domains.json]: This file should be removed. Consider it removed.

File: cypress/integration/auction-offering.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+          .contains('Your bid is winning this auction!')
+          .should('not.exist')
+
+        cy.switchAccount(2)
+        findListItemByText(url)
+          .contains('Your bid is winning this auction!')
+          .should('exist')
+      })
+
+      // TODO: test auction finalization
```
Comment (ladislavdubravsky) [inline cypress/integration/auction-offering.js:99]: This deserves an issue, since it's in the most basic workflows. https://github.com/vacuumlabs/name-bazaar/issues/48

**Comment by ladislavdubravsky** [comment]
I got
```
Timed out retrying after 15000ms: Expected to find content: '1.7' within the element: [ <div.price>, 1 more... ] but never did.
cypress/integration/auction-offering.js:85:35
  83 |         findListItemByText(url).within(() => {
  84 |           cy.get(':nth-child(3)').contains('2') // total bids
> 85 |           cy.get(':nth-child(5)').contains('1.7') // highest bid
     |                                   ^
  86 |         })
  87 | 
  88 |         cy.switchAccount(1)
```
twice, but if it passes for you, it's probably just the slowness.

**Review by ladislavdubravsky — APPROVED** [review]
Great job with the tests! It'll be much less tiring now :)
